project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.info

```diff
@@ -30,12 +30,12 @@ features[flag][] = subscribe_c4m_follow_node
 features[flag][] = subscribe_c4m_follow_term
 features[og_features_permission][] = node:group:invite any user
 features[og_features_permission][] = node:project:invite any user
+features[user_permission][] = flag subscribe_c4m_follow_comment
```

Comment:
I want to believe that whoever export this feature didn't do it manually, hence it seems like a mismatch between machines configuration and it is a never ending game.
Please revert this to decrease the diff of this PR, there is no point to commit this now.